test_scripts/UpdateVideoStreamingCapabilities/common.lua

```diff
@@ -241,4 +325,17 @@ function m.stopVideoService(pAppId)
     end)
 end
 
+function m.preloadedPTUpdate()
+  local pt = actions.sdl.getPreloadedPT()
+  pt.policy_table.app_policies["default"].encryption_required = false
+  pt.policy_table.functional_groupings["Base-4"].encryption_required = false
+  pt.policy_table.functional_groupings["DataConsent-2"].rpcs = json.null
+  actions.sdl.setPreloadedPT(pt)
+end
+
+function m.preconditions()
```

Comment:
@GetmanetsIrina I would propose to create separate function and use it in resolution switching scripts but use `actions.preconditions` in other scripts through common module.

Reply:
Requiring of `sslHandshakeFlow` redefines the origin `actions.preconditions`, so this precondition should be used for all scripts because of requiring of `sslHandshakeFlow` file.
This is some rollback of redundant precondition, without this precondition all RPCs from `Base-4` should be encrypted, or `sslHandshakeFlow` should be required only in required scripts and in this case, some functions will be moved locally in scripts.